autotest/ogr/ogr_mitab.py

```diff
@@ -416,14 +388,8 @@ def test_ogr_mitab_13():
 # Verify that field widths and precisions are propagated correctly in MIF.
 
 
+@pytest.mark.usefixtures('gml_ds')
```

Comment:
This is equivalent to passing the `gml_ds` fixture as the argument to the test function, except that passing it as an argument causes some linters to complain about the function argument shadowing the imported `gml_ds` value (and it _needs_ to be imported for pytest to find it as a fixture). Hence I used the usefixtures marker instead